src/jit/gentree.h

```diff
@@ -4037,26 +3887,186 @@ struct GenTreeIndir : public GenTreeOp
 #endif
 };
 
+// gtBlk  -- 'block' (GT_BLK, GT_STORE_BLK). */
+struct GenTreeBlk : public GenTreeIndir
+{
+public:
+    // The data to be stored (null for GT_BLK)
+    GenTree*& Data()
+    {
+        return gtOp2;
+    }
+    void SetData(GenTree* dataNode)
+    {
+        gtOp2 = dataNode;
+    }
+
+    // The size of the buffer to be copied.
+    unsigned Size()
+    {
```

Comment:
Done.

---
In reply to: [77248111](https://github.com/dotnet/coreclr/pull/7027#discussion_r77248111) [](ancestors = 77248111)